modules/developers-guide/pages/graphql.adoc

```diff
@@ -1,6 +1,109 @@
 = GraphQL API
 
-The Stargate GraphQL API is implemented to use GraphQL mutations and queries with any Cassandra deployment.
+== Overview
 
-* xref:quickstart:quick_start-graphql.adoc[GraphQL QuickStart]
-* xref:graphql-using.adoc[Using the GraphQL API]
+The Stargate GraphQL API is implemented to easily modify and query your table data
+using GraphQL types, mutations, and queries with any Cassandra deployment.
+
+The Stargate GraphQL API has two orientations, one developed from GraphQL principles, and one developed with the Cassandra Query Language (CQL)
```

Comment:
Rather than "orientations" what about "modes" (e.g. "GraphQL API two modes, one being CQL first and the other driven by native GraphQL schema")? Would that be more obvious for the tech inclined and potentially translate better?

Another option might be "emphasis". As in "GraphQL API has two main emphasis, one driven by GraphQL native schema and the other by CQL schema"

Reply:
changed to cql-first and graphql-first